systems/plants/@RigidBodyManipulator/RigidBodyManipulator.m

```diff
@@ -917,10 +917,10 @@
     
     function body_id = findLinkId(model,linkname,robot,error_level)
       % @param robot can be the robot number or the name of a robot
-      % robot=0 means look at all robots
+      % robot<0 means look at all robots
       % @param error_level >0 for throw error, 0 for throw warning, <0 for do nothing. @default throw error
       % @ingroup Kinematic Tree
-      if nargin<3 || isempty(robot), robot=0; end
+      if nargin<3 || isempty(robot), robot=-1; end
```

Comment:
Here are the other methods I could find that take a `robotnum` argument and may need to be changed:
* `getCOM` - The logic here should be updated to match that in `getMass`
* `forwardJacDot` - This should probably just be passing the `robotnum` into `getCOM` rather than duplicating code in ll. 32 - 48
* `getPositionFrame` - Currently this will throw an indexing error if passed `robotnum` of 0 or -1
* `getVelocityFrame` - Same
*  `getStateFrame` - Same